test/e2e/handler/states.go

```diff
@@ -178,6 +178,29 @@ func vlanUpWithStaticIP(iface string, ipAddress string) nmstate.State {
 `, iface, ipAddress))
 }
 
+func resetPrimaryAndSecondaryNICS() nmstate.State {
```

Comment:
I'm not super sure about the conventions, but I think the last s should be small

Reply:
Do you mean `resetPrimaryAndSecondaryNics`